tests/test_data.py

```diff
@@ -460,5 +460,21 @@ def test_get_pulse_times(self):
         self.assertTrue(np.all(test_03_times == pulse_times))
         self.assertTrue(np.all(test_03_dc == pulse_dc))
 
+class Backup(unittest.TestCase):
+
+    def test_move_files(self):
+        source_dir = [['tests/data/test_source/dir_1'],['tests/data/test_source/dir_2']]
+        #source_files = [os.path.join(source_dir,'load_hdf_contents_test.hdf'), os.path.join(source_dir,'load_hdf_group_test.hdf')]
+
+        dest_dir = 'tests/tmp/test_destination'
+        if not os.path.exists(dest_dir):
+            os.mkdir(dest_dir)
```

Comment:
you should also delete the destination file if it already exists

Reply:
do you mean for the unit tests?

Reply:
I have added code to empty our test destination prior to running unittest.